pkg/workload/histogram/histogram.go

```diff
@@ -161,6 +164,9 @@ func makePrometheusLatencyHistogramBuckets() []float64 {
 }
 
 func (w *Registry) getPrometheusHistogram(name string) prometheus.Histogram {
+	// Metric names must be sanitized or NewHistogram will panic.
+	name = invalidPrometheusMetricRe.ReplaceAllString(name, "_")
```

Comment:
This puts a regexp replace into the hot path of workloads, see for example

https://github.com/cockroachdb/cockroach/blob/31847acd14ed27a340dfc620a544c3e33cbd7c9a/pkg/workload/kv/kv.go#L404-L406

so we should get to a place where we sanitize only once, when the histogram is created. I think this is straightforward: only sanitize the `Name` field in the `if !ok` branch below, and leave it alone here.